api/src/integrations/core/permit-utils.js

```diff
@@ -109,6 +109,17 @@ class Permits extends BaseRecordUtils {
 
     return permits;
   }
+
+  getPermitType(typeCode) {
+    switch(typeCode){
+      case 'OGP':
+        return 'Permit';
```

Comment:
We reference 'Permit' here and 'Original Permit' for the same case it would be best to stick to one name but there may be a reason for that.

Also is there an opportunity to refactor this switch into the function suggested above?